extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamFrames.java

```diff
@@ -0,0 +1,113 @@
+package org.schabi.newpipe.extractor.stream;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class StreamFrames {
```

Comment:
This should probably be just a class wrapping around extracted data (see below).